data_science/terraform/dlami_asg/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "key_name" {}
+variable "vpc_id" {}
+
+variable "hashed_password" {
+  default = "sha1:5310f21e370d:a4d66e725c179218638c21c03d83933aa066db2d"
```

Comment:
It would be useful to have a comment explaining how to generate this variable, _or_ declare the plaintext password as a Terraform variable, and do the SHA1 hashing at plan-time.